jest.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  moduleNameMapper: {
+    "^vue$": "vue/dist/vue.common.js"
+  },
+  moduleFileExtensions: ["js", "json", "vue"],
+  transform: {
+    ".*\\.(vue)$": "vue-jest",
+    "^.+\\.js$": "babel-jest"
+  },
+  collectCoverage: true,
+  collectCoverageFrom: [
+    "lib/**/*.js",
+    "public/js/**/*.{js,vue}",
+    "!public/js/dist/**",
+    "!**/node_modules/**"
+  ],
+  roots: ["lib", "public/js"],
+  setupFiles: ["./lib/testSetup.js"],
+  testURL: "http://localhost/",
+  globals: {
+    SIMULATING: process.env.SIMULATING === "true" ? true : false
```

Comment:
not a huge deal, but this would have the same effect with less words: 
```suggestion
    SIMULATING: process.env.SIMULATING === "true"
```

Reply:
Forgot to mention @underscoredotspace, thanks for this feedback! I updated this in my most recent commit.